arrayMethodsLab.js

```diff
@@ -0,0 +1,308 @@
+// # Exercises
+
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+// ```
+console.log("Question 1: ")
+let array1 = [1, 2, 3, 4, 5];
+let output1 = array1.map( (currentElem) => {
+    return currentElem+=10;
+});
+console.log(output1);
+console.log("------------------------------------------------------------")
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+
+// ```
+// Input: ["a", 123, "b", "c", {name: "cat"}]
+// Output: ["a", "", "b", "c", ""]
+// ```
+console.log("Question 2: ")
+let array2 = [ "a", 123, "b", "c", {name: "cat"} ];
+let output2 = array2.map( (currentElem) => {
+    if(typeof currentElem !== 'string') {
+        return "";
+    } else {
+        return currentElem;
+    }
+} )
+console.log(output2);
+console.log("------------------------------------------------------------")
+// 3. Write a function that returns only the even elements in an array of numbers.
+
+// ```
+// Input: [1,2,3,4,5]
+// Output: [2,4]
+// ```
+console.log("Question 3: ")
+let array3 = [1, 2, 3, 4, 5];
+const checkIfEven = (i) => {
+    return i % 2 === 0
+} 
+let output3 = array3.filter(checkIfEven);
+console.log(output3);
+console.log("------------------------------------------------------------")
```

Comment:
you have the right idea. but we want you to use the array method in the function . 

const checkIfEven = (array3) => {
	    return array3.filter((i) => {
return i % 2 === 0
	})}